app/src/main/java/xyz/kotlout/kotlout/model/experiment/BinomialExperiment.java

```diff
@@ -9,9 +9,8 @@
  */
 public class BinomialExperiment extends Experiment {
 
-  private List<BinomialTrial> trials;
-
   public ExperimentType type = ExperimentType.BINOMIAL;
```

Comment:
Should we consider making these types `static`? Or would that lead to problems with the firebase integration?
Also shouldn't this be defined in the base class and overloaded here or something?

Because I don't really see any reason that these are runtime fields.

Reply:
You can't "overload" properties. The superclass' version of trial will just get hidden by the subclass, making it basically useless.

Static fields are shared across all instances of the class, so maybe it could be done for the `ExperimentType type` property, but not trials.

Also, you're reviewing removed lines in this review, so I couldn't easily tell what you were referring to.

Reply:
Not actually related to this pr. Nvm.